topiary-core/benches/benchmark.rs

```diff
@@ -5,17 +5,17 @@ use std::io;
 use topiary_core::{formatter, Language, Operation, TopiaryQuery};
 
 async fn format() {
-    let input = fs::read_to_string("../topiary-cli/tests/samples/input/ocaml.ml").unwrap();
-    let query_content = fs::read_to_string("../topiary-queries/queries/ocaml.scm").unwrap();
-    let ocaml = tree_sitter_ocaml::LANGUAGE_OCAML;
+    let input = fs::read_to_string("../topiary-cli/tests/samples/input/json.json").unwrap();
+    let query_content = fs::read_to_string("../topiary-queries/queries/json.scm").unwrap();
+    let json = tree_sitter_json::LANGUAGE;
```

Comment:
performance issues with the ocaml grammar previously discussed in https://github.com/tweag/topiary/pull/859 suggest that swapping the benchmark language to (anything) JSON will reduce time spent in CI.

Reply:
Does that make it a good test, though? JSON formatting is quite trivial, whereas OCaml's is really hitting a large surface area of Topiary. My hunch would be therefore, that for benchmarking, pushing JSON through would result in a weaker signal.

(That said: We are not currently making any use of the benchmarking results, so the above point is kind-of-academic compared to the usefulness of reduced CI time.)